pkg/apis/networking/validation/validation.go

```diff
@@ -239,8 +239,7 @@ func validateIngress(ingress *networking.Ingress, opts IngressValidationOptions,
 // ValidateIngressCreate validates Ingresses on create.
 func ValidateIngressCreate(ingress *networking.Ingress, requestGV schema.GroupVersion) field.ErrorList {
 	allErrs := field.ErrorList{}
-	var opts IngressValidationOptions
-	opts = IngressValidationOptions{
+	var opts = IngressValidationOptions{
```

Comment:
nit.

```suggestion
	opts := IngressValidationOptions{
```

Here and below.